xml/ha_config_cli.xml

```diff
@@ -217,8 +232,24 @@
      </para>
     </listitem>
     <listitem>
+      <formalpara>
+        <title>As &crmsh; Cluster Scripts:</title>
+        <para>These are a collection of metadata, references to RPM packages,
+          configuration files, and &crmsh; subommands bundled under a single,
+          yet descriptive name. These are managed through the
+          <command>crm script</command> command.
+        </para>
+      </formalpara>
+      <para>Do not confuse them with crm shell scripts: although both share
+        some common objectives, the crm shell scripts just contains subcommands
+        whereas cluster scripts incorporate much more and goes beyond simple
```

Comment:
typo: goes -> go 
I'm not sure about the "goes beyond" part -> maybe remove it?